addons/mass_mailing/static/tests/mass_mailing_favourite_filter_tests.js

```diff
@@ -407,5 +407,72 @@ QUnit.module('favorite filter widget', (hooks) => {
         assert.isNotVisible(fixture.querySelector('.o_mass_mailing_remove_filter'),
             "should not have option to remove filter because mailing domain is changed");
     });
+
+    QUnit.test('mailing_model_name can be empty without throwing an error', async (assert) => {
```

Comment:
Define a more 'realistic' usecase, and move this so that it's in alphabetical order based on the name of the tests
```suggestion
    QUnit.test('filter widget works in edit and readonly', async (assert) => {
```

Reply:
Ah, my bad. I guess they aren't. Best to keep them together by 'theme' if possible then, otherwise it's fine.